### ISSUE TITLE
Timeline week styling - bug? - wrong columns css

### ISSUE THREAD

**Issue by heiovsky** [almende/vis#3445]
Week 39 column background should be gray to the red line.

![image](https://user-images.githubusercontent.com/17221757/30368486-91017082-9871-11e7-9602-1002842197c6.png)

**Comment by marcortw** [comment]
Yes, it's a bug (known to me, I initially wrote the 'week styling' feature). There's a problem and some complexity here with the way how the major and minor gridlines are calculated and drawn. I could probably fix this, but it will take me weeks until I find the time for it. Nevertheless, I volunteer -
 @yotamberk could you please assign that issue to me?

**Comment by wimrijnders** [comment]
@marcortw I'm not @yotamberk but I'd be happy to assign you, however you're not on the list of assignees. I think it's a maintainer thing, are you a maintainer? But I believe you can totally work on it regardless.
